src/main/cc/proxy_client/link_command_inputs.cc

```diff
@@ -0,0 +1,61 @@
+#include "src/main/cc/proxy_client/link_command_inputs.h"
+
+#include <set>
+#include <string>
+#include <sys/stat.h>
+
+namespace remote_client {
+
+using std::set;
+using std::string;
+
+void FindLibrarySearchPaths(int argc, char** argv, set<string>* lib_search_paths) {
+	for (int i = 0; i < argc; ++i) {
```

Comment:
nit: please use consistent indent size with the rest of the code

Reply:
Done.

Reply:
I don't see the change, but please use 2 spaces for indent before submitting. Thank you!

Reply:
Sorry, it seems I set "Indent using 2 spaces" but didn't actually convert the existing lines, I did that now.